### PR TITLE
dev branch > master on forked repo

### PR DESCRIPTION
**Pull request by joshband**

### CODE REVIEW

**Review by doubleminus — APPROVED** [review]
LGTM.

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ node_modules
 *.ipr
 *.iws
 /.idea
+myConfig.js
```
Comment (magic-madrigal) [inline .gitignore:8]: Why did we add this to the .gitignore files? What does this file drive?

File: examples/profile-createOrUpdate.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+var myConfig = require('../myConfig');
```
Comment (magic-madrigal) [inline examples/profile-createOrUpdate.js:1]: I don't see this file documented in the read me. Looks like it's a file that could be used to store creds, but do you feel we should be explaining that?

**Review by magic-madrigal — COMMENTED** [review]
The code looks good and I love the examples. Since this is a public project I think we need to be also updating the README.MD to reflect the changes we made.

Also, the myConfig.js files could be confusing for someone using the module for the first time.
Is this file referenced in other examples? Maybe a simple comment calling out what it's purpose is and format should be. Think of the lowest common denominator.

File: examples/profile-createOrUpdate.js (hunk 2 above)
Comment (joshband) [reply examples/profile-createOrUpdate.js:1]: I'll update the docs to reflect that this is the creds file and link it back to exampleConfig (which is included)

File: .gitignore (hunk 1 above)
Comment (joshband) [reply .gitignore:8]: My creds for waterbears-dev are included in this file

**Pull request merged**